src/AnnotatedCode.c

```diff
@@ -73,3 +101,63 @@ R_API void r_annotated_code_print_json(RAnnotatedCode *code)
 	r_cons_printf ("%s\n", pj_string (pj));
 	pj_free (pj);
 }
+
+R_API void r_annotated_code_print_with_syntax_highlighting(RAnnotatedCode* code, RCore *core)
+{
+    if (code->annotations.len == 0) {
+        r_cons_printf("%s\n", code->code);
+        return;
+    }
+
+
+    size_t cur = 0;
+    size_t len = strlen(code->code);
+
+	RCodeAnnotation *annotation;
+	r_vector_foreach (&code->annotations, annotation) {
+        if (annotation->type != R_CODE_ANNOTATION_TYPE_SYNTAX_HIGHLIGHT)
```

Comment:
it's a good practice always to use curly braces, even with a single line condition/loop
https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/

Reply:
No it's not, I explicitly allow it for r2ghidra ;-P

Reply:
Sorry nevermind, the AnnotatedCode.c file is meant to be merged into r2 at some point, so there should be no single-line ifs here. But in the rest of r2ghidra it is allowed.